ui/GNUmakefile

```diff
@@ -25,6 +25,8 @@ deps: clean
 	cd packages/consul-ui && \
 		$(MAKE) deps
 
+display-node-version:
+	node --version
```

Comment:
@valeriia-ruban can we remove this  and the call to it on line 40. This was left in from when I was testing what was going on with Vercel builds failing.